app/views/user/_exercises_table.erb

```diff
@@ -18,7 +18,7 @@
       <th>Language</th>
       <th>Exercise</th>
       <th>Iterations</th>
-      <th data-sorted="true" data-sorted-direction="descending">Latest Iteration</th>
+      <th data-sorted="true" data-sorted-direction="ascending">Latest Iteration</th>
```

Comment:
I agree that line 26 is where the list is created, but line 21 changed the default sort order.  We are trying for a so called "descending" order, in our minds, but the list that is being sorted is in an arbitrary direction, and we simply want it to be the other direction.  The name "descending" vs. "ascending" is hidden, to the webpage user.  So, I proposed what I thought would be the simplest method to result in the desired outcome.  

On the other, hand, changing the order the list is created in line 26 will also result in the opposite display order on the web page.